lib/features/announcement/ui/event_list/event_page.dart

```diff
@@ -0,0 +1,272 @@
+                          fontSize: 24,
+                        ),
+                      ),
+                      const SizedBox(height: 8),
+                      const Text(
+                        'testingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtesting',
+                        style: TextStyle(
+                          color: Color(0xFF0A1931),
+                          fontSize: 16,
+                          height: 1.5,
+                        ),
+                      ),
+                      const SizedBox(height: 24),
+                      ElevatedButton(
+                        onPressed: () {
+                          // Handle button press to register for the event
+                        },
+                        style: ElevatedButton.styleFrom(
+                          padding: const EdgeInsets.symmetric(
+                            horizontal: 24,
+                            vertical: 12,
+                          ),
+                          shape: RoundedRectangleBorder(
+                            borderRadius: BorderRadius.circular(24),
+                          ),
+                          backgroundColor: const Color(constants.primaryDarkTeal),
+                        ),
+                        child: Row(
+                          mainAxisSize: MainAxisSize.min,
+                          children: const [
+                            Icon(
+                              Icons.event,
+                              color: Colors.white,
+                            ),
+                            SizedBox(width: 8),
+                            Text(
+                              'Register for Event',
+                              style: TextStyle(
+                                color: Colors.white,
+                                fontWeight: FontWeight.bold,
+                                fontSize: 16,
+                                fontFamily: 'Montserrat', // Custom font family
+                              ),
+                            ),
+                          ],
+                        ),
+                      ),
+                      const SizedBox(height: 16),
+                      Padding(
+                        padding: const EdgeInsets.symmetric(horizontal: 16),
+                        child: Row(
+                          children: [
+                            Expanded(
+                              child: Container(
+                                height: 2,
+                                color: Colors.grey[200],
+                              ),
+                            ),
+                            const SizedBox(width: 16),
+                            const Text(
+                              'Share on social media',
```

Comment:
just a placeholder?  no need to fix as a resource file entry

Reply:
done